### DISCUSSION TITLE
multiple matchers

### DISCUSSION THREAD

**Discussion by 0xspade** [projectdiscovery/httpx#934] [Q&A] [answered]
Is it possible to put a multiple matchers for example:

`httpx -mr '<some-regex>' -ms '<some-string>' -mr '<some-regex-again>' -ms 'some-string-again'`

is this possible? haven't tried this yet but better to ask first :)

**Comment by olearycrew** [comment, marked as answer]
@0xspade looks possible to me:

`httpx -u http://boleary.dev -ms 'brendan' -ms 'olearycrew'`

<img width="996" alt="CleanShot 2023-04-19 at 11 56 37@2x" src="https://user-images.githubusercontent.com/6044920/233132571-6411fb3d-c8b7-48c8-a7d5-f2f771adbc97.png">

`httpx -u http://boleary.dev -ms 'brendan' -ms 'olearycrew' -ms 'forsurenotonmywebsite'`

<img width="1311" alt="CleanShot 2023-04-19 at 11 57 17@2x" src="https://user-images.githubusercontent.com/6044920/233132730-dee8fbba-3531-432e-a20f-bf80c32a30ff.png">
